docker/Dockerfile

```diff
@@ -74,4 +78,4 @@ RUN cd /torchfort && mkdir build && cd build && \
 ENV LD_LIBRARY_PATH /opt/torchfort/lib:${LD_LIBRARY_PATH}
 ENV LD_LIBRARY_PATH /usr/local/lib/python3.10/dist-packages/torch/lib:${LD_LIBRARY_PATH}
 
-ENTRYPOINT bash
+#ENTRYPOINT bash
```

Comment:
Is this entry point problematic for you?

Reply:
I am generally not a big fan of entry points because they are counterintuitive for me. But I think we can leave it here.

Reply:
fixed